Packs/qualys/ReleaseNotes/1_2_3.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Qualys v2
+- Command ***qualys-host-list***: Added optional arguments ***details***
```

Comment:
you need to update this to follow one of the prefixes: https://xsoar.pan.dev/docs/documentation/release-notes#general